microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -149,53 +146,42 @@ The experiments checkboxes that appear in the latest version of the Canary previ
 ## Allow extensions to load custom stylesheets
 <!-- checkbox has a (?) link to this anchor wording -->
 
-Some Microsoft Edge Add-ons can define custom color themes for DevTools. If you install an add-on with a theme, you need to enable the **Allow extensions to load custom stylesheets** experiment to view the add-on themes.
+Some Microsoft Edge Add-ons can define custom color themes for DevTools. If you install an add-on that defines a custom color theme for DevTools, you need to enable the **Allow extensions to load custom stylesheets** experiment to view the add-on themes.
+
+See also:
+* [Apply a color theme to DevTools](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/theme)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is present in Microsoft Edge Canary 124.
+* This checkbox is present in Microsoft Edge Stable 122.
 
 
 <!-- ====================================================================== -->
 ## Capture node creation stacks
 <!-- checkbox has a (?) link to this anchor wording -->
 
-To capture JavaScript stack traces when DOM nodes are added to the DOM at runtime, enable this experiment. The captured stack traces are displayed in the **Stack Trace** pane of the **Elements** panel.
-
-Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
-
-
-<!-- ====================================================================== -->
-## Automatically pretty print minified sources
-<!-- no (?) link -->
-
-<!-- as of 115, not 114-, checkbox label is now:
-Automatically pretty print minified sources
-instead of:
-Automatically pretty print in the Sources Panel
--->
+To capture JavaScript stack traces when DOM nodes are added to the DOM at runtime, enable this experiment.  The captured stack traces are displayed in the **Stack Trace** pane of the **Elements** tool.<!-- todo: what tool contains this Stack Trace pane?  Sources tool?  Crash Analyzer tool?  Console?  where in that tool?  Update See Also accordingly -->
```

Comment:
What tool contains this Stack Trace pane? Sources tool? Crash Analyzer tool? Console? where in that tool?
@captainbrosset 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-3077#capture-node-creation-stacks

Reply:
The **Stack Trace** pane is in the **Elements** tool, in the sidebar, next to **Styles**, **Computed**, etc.

![image](https://github.com/MicrosoftDocs/edge-developer/assets/1152698/a1ad530d-3bff-4520-b5b9-04c0ea4ed8e8)

After you enable the experiment, here's how to test:

1. Open the Elements tool
2. Use a webpage that creates DOM nodes dynamically with JS, such as the demo-to-do app
3. Create a new TODO item
4. Select the new TODO item with the inspect tool
5. Open the **Stack Trace** pane.